### PR TITLE
Add serial number and revision number to svid minting log entries

### PR DESCRIPTION
**Pull request by alexviktorov**
**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [x] Proper tests/regressions included?
- [x] Documentation updated?

**Affected functionality**
Log entries will include extra fields; datetime of Svid minting audit logs changed format from Unix to RFC3339 

**Description of change**
Added serial number and revision number to svid minting log entries: both for Batch/NewX509Svid and only serial number for MintX509Svid

**Which issue this PR fixes**
#3688

### CODE REVIEW

File: pkg/server/api/svid/v1/service.go [hunk 1]
```diff
@@ -115,11 +115,12 @@ func (s *Service) MintX509SVID(ctx context.Context, req *svidv1.MintX509SVIDRequ
 	}
 
 	rpccontext.AddRPCAuditFields(ctx, logrus.Fields{
-		telemetry.ExpiresAt: x509SVID[0].NotAfter.Unix(),
+		telemetry.ExpiresAt: x509SVID[0].NotAfter.Format(time.RFC3339),
```
Comment (evan2645) [inline pkg/server/api/svid/v1/service.go:118]: Was this change intentional? I don't have a strong opinion on the formatting here, the concern from my end is more around backward compatibility for folks using this field today

File: pkg/server/api/svid/v1/service.go [hunk 2]
```diff
@@ -115,11 +115,12 @@ func (s *Service) MintX509SVID(ctx context.Context, req *svidv1.MintX509SVIDRequ
...
 	})
 
 	rpccontext.AuditRPCWithFields(ctx, commonX509SVIDLogFields)
 	log.WithField(telemetry.Expiration, x509SVID[0].NotAfter.Format(time.RFC3339)).
+		WithField(telemetry.SerialNumber, x509SVID[0].SerialNumber).
```
Comment (evan2645) [inline pkg/server/api/svid/v1/service.go:123]: I think this is an x509.Certificate type, and the SerialNumber field there is actually *big.Int which is a struct. I think we need to call the String() method to get something printable

```suggestion
		WithField(telemetry.SerialNumber, x509SVID[0].SerialNumber.String()).
```

**Review by evan2645 — COMMENTED** [review]
Thanks so much for this contribution @alexviktorov! Couple minor comments, should be good to go after that.

File: pkg/server/api/svid/v1/service.go (hunk 1 above)
Comment (alexviktorov) [reply pkg/server/api/svid/v1/service.go:118]: Yes, just a couple of lines below on line 122  the Expiration is logged in time.RFC3339 format so changed it for consistency's sake after discussing with Ryan. I don't have a strong opinion though, I can easily revert it.

Comment (evan2645) [reply pkg/server/api/svid/v1/service.go:118]: Ah gotcha ... alrighty then.

**Review by evan2645 — APPROVED** [review]
Thank you @alexviktorov!
